mod_reforged/hooks/skills/perks/perk_nine_lives.nut

```diff
@@ -0,0 +1,40 @@
+::mods_hookExactClass("skills/perks/perk_nine_lives", function(o) {
+    local create = o.create;
+	o.create = function()
+	{
+        create();
+
+        // New Variables to control and tweak the behavior of this perk
+        this.m.MinHP <- 11;
+        this.m.MaxHP <- 15;
+
+        // This makes the perk now always show by default on top of characters while it is not procced yet
+		this.m.IconMini = "perk_07_mini";
+		this.m.Overlay = "perk_07";
```

Comment:
`this.m.Overlay` is unnecessary here - as it is meant for triggering an icon on the character when this skill is added to the character. We don't need that in this case, so it's better to remove it for clarity.